src/main/java/org/jenkinsci/plugins/workflow/actions/ArgumentsAction.java

```diff
@@ -110,6 +110,15 @@ public static boolean isOverSized(@CheckForNull Object o, final int maxElements)
         return false;
     }
 
+    /**
+     * Check for single oversized fields much like {@link #isOverSized(Object, int)} but using {@link #MAX_RETAINED_LENGTH}.
+     * @param o Object to check for being oversized or holding an oversized value.
+     * @return True if object contains an oversized input, else false.
+     */
+    protected static boolean isOversized(@CheckForNull Object o) {
```

Comment:
Still recommend these methods, and the constant, be removed from the API and moved to `ArgumentsActionImpl`.